package.json

```diff
@@ -8,7 +8,7 @@
     "Adam Rudd <damvrr@gmail.com>"
   ],
   "scripts": {
-    "test": "tape test.js | faucet",
+    "test": "tape test.js | faucet && standard",
     "ci": "tape test.js && node testRandom"
```

Comment:
Tap-spec seems to be working as well. Where do you want to see standard as well? In the CI script for Travis?

I can push both if you'd like me to.